cli/command/system/version.go

```diff
@@ -189,3 +211,32 @@ func getDetailsOrder(v types.ComponentVersion) []string {
 	sort.Strings(out)
 	return out
 }
+
+func getKubernetesVersion(dockerCli command.Cli, cmd *cobra.Command) (*kubernetesVersion, error) {
+	if !dockerCli.ClientInfo().HasKubernetes() {
+		return nil, nil
+	}
+
+	kli, err := kubernetes.WrapCli(dockerCli, cmd)
+	if err != nil {
+		return nil, err
+	}
+
+	apiVersion, err := kli.GetAPIVersion()
+	if err != nil {
```

Comment:
If this fails it should still attempt to get the kube version, so maybe this should be split into two functions.

Reply:
Right, done